hca/cli.py

```diff
@@ -107,12 +107,14 @@ def help(args):
 
 
 def main(args=None):
+    if not args:
+        args = sys.argv
     if '--help' in sys.argv or '-h' in sys.argv:
```

Comment:
Could you change these refs to `sys.argv` to `args`?

Reply:
addressed